lib/benefits_intake_service/service.rb

```diff
@@ -24,6 +24,15 @@ class Service < Common::Client::Base
 
     REQUIRED_CREATE_HEADERS = %w[X-VA-First-Name X-VA-Last-Name X-VA-SSN X-VA-Birth-Date].freeze
 
+    # Validate a file satisfies Benefits Intake specifications. File must be a PDF.
+    # @param [String] doc_path
+    def validate_document(pdf_path:)
```

Comment:
@wayne-weibel Based on your comment, this should be `doc_path:` instead of `pdf_path:`

Reply:
updated.  thanks!